payments-core/src/test/java/com/stripe/android/view/CardWidgetViewModelTest.kt

```diff
@@ -61,6 +64,7 @@ class CardWidgetViewModelTest {
             paymentConfigProvider = { paymentConfig },
             stripeRepository = stripeRepository,
             dispatcher = testDispatcher,
+            handle = SavedStateHandle()
         )
```

Comment:
Can we add a test to ensure it's saving to the handle?

Reply:
Done 👍

Reply:
Done 👍